### PR TITLE
added group expenses schema to the database

### PR DESCRIPTION
**Pull request by clintBock**
be gentle niko

### CODE REVIEW

File: database/migrations/2017_03_14_030738_CreateGroupMemberTable.php [hunk 1]
```diff
@@ -17,6 +17,9 @@ public function up()
             $table->increments('id');
             $table->unsignedInteger('group_id');
             $table->string('username');
+            $table->boolean('status_code');
```
Comment (nikoootine) [inline database/migrations/2017_03_14_030738_CreateGroupMemberTable.php]: status_code is a string. its either Accepted, Rejected, Pending, etcc

**Comment by clintBock** [comment]
i accidentally pushed nothing to the branch that was cool, but it's there now

File: database/migrations/2017_03_24_014813_CreateGroupExpensesTable.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('groupexpenses', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('group_id');
+            $table->unsignedInteger('amount');
```
Comment (nikoootine) [inline database/migrations/2017_03_24_014813_CreateGroupExpensesTable.php]: amount should not be an unsigned integer. float or decimal or double or i dont know

File: database/migrations/2017_03_24_014813_CreateGroupExpensesTable.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            $table->string('comments');
+            $table->dateTime('date_added');
```
Comment (nikoootine) [inline database/migrations/2017_03_24_014813_CreateGroupExpensesTable.php]: should be timestamp

File: database/migrations/2017_03_24_020639_UpdateGroupExpensesTable.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('groupexpenses', function (Blueprint $table) {
+            $table->unique('group_id');
```
Comment (nikoootine) [inline database/migrations/2017_03_24_020639_UpdateGroupExpensesTable.php]: shouldnt be unique. a group can have as much as expenses as they want

File: database/migrations/2017_03_24_021429_CreateGroupSharedExpensesTable.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+     */
+    public function up()
+    {
+        Schema::create('groupsharedexpenses', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('group_expense_id');
+            $table->string('username');
+            $table->float('amount_owed');
+            $table->string('comments');
+            $table->date('date_added');
```
Comment (itsmattli) [inline database/migrations/2017_03_24_021429_CreateGroupSharedExpensesTable.php]: HEY FARMER, you forgot date_settled

**Pull request merged**